WordPress/Classes/ViewRelated/Reader/ReaderPostTableViewCell.m

```diff
@@ -55,6 +46,23 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
     return self;
 }
 
+- (UIView *)newSideBoarderView
+{
+    UIView *view = [[UIView alloc] init];
+    view.translatesAutoresizingMaskIntoConstraints = NO;
+    view.backgroundColor = [UIColor colorWithRed:210.0/255.0 green:222.0/255.0 blue:238.0/255.0 alpha:1.0];
```

Comment:
UIColor? Probably.  This was an existing snipped that I moved around.  We probably already have that color in the style guide.  I'll update to reference it from there.  Also that naming... "boarder" should be "border" will update that as well.